cloud/resource_apikey.go

```diff
@@ -76,28 +67,39 @@ func resourceApiKey() *schema.Resource {
 			"organization": {
 				Type:         schema.TypeString,
 				Required:     true,
+				ForceNew:     true,
 				Description:  descriptions["organization"],
 				ValidateFunc: validateNotBlank,
 			},
 			"name": {
 				Type:         schema.TypeString,
 				Required:     true,
+				ForceNew:     true,
 				Description:  descriptions["apikey_name"],
 				ValidateFunc: validateNotBlank,
 			},
 			"instance_name": {
 				Type:        schema.TypeString,
 				Required:    true,
+				ForceNew:    true,
 				Description: descriptions["instance_name"],
 			},
+			"token": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Sensitive:   true,
+				Description: descriptions["token"],
+			},
```

Comment:
This is a computed field, but doesn't seem to actually be computed (as done in the data source) https://github.com/streamnative/terraform-provider-streamnative/blob/8614ecf2b8ac588952de038f351ba35b2a92c0ea/cloud/data_source_apikey.go#L172